types/noble/index.d.ts

```diff
@@ -9,9 +12,13 @@
 import events = require("events");
 
 export declare function startScanning(): void;
+export declare function startScanning(callback: Function): void;
 export declare function startScanning(serviceUUIDs: string[]): void;
+export declare function startScanning(serviceUUIDs: string[], callback: Function): void;
 export declare function startScanning(serviceUUIDs: string[], allowDuplicates: boolean): void;
+export declare function startScanning(serviceUUIDs: string[], allowDuplicates: boolean, callback: Function): void;
 export declare function stopScanning(): void;
+export declare function stopScanning(callback: Function): void;
```

Comment:
:+1: I thought I read that was a bad practice in the FAQ, but I realized that was only for getters/setters.